numpy/core/function_base.py

```diff
@@ -67,6 +67,13 @@ def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None,
 
         .. versionadded:: 1.16.0
 
+    keepdims : bool, optional
+        If this is set to True, the ndarray returned will have the same
+        dimension as that of start and end.  Also, the value of dimension of
+        star and end corresponding to axis must have value 1.
+
+        .. versionadded:: 1.18.0
```

Comment:
In the likely even this misses this release (don't comit this change until 1.18 is out without this PR, just in case :))

```suggestion
        .. versionadded:: 1.19.0
```